deltas/tokenizers/tokenizer.py

```diff
@@ -58,3 +75,63 @@ def _tokenize(self, text, token_class=None):
                 tokens[value] = token
 
             yield token
+
+
+class TokenProcessor:
+    def segment(self, tokenized_text):
```

Comment:
Should be more general.  e.g. "process"

Reply:
fixed - I changed it to process everywhere